plugins/powerpc/powerpc_arith.ml

```diff
@@ -0,0 +1,420 @@
+
+open Powerpc.Std
+
+(** Fixed-Point Arithmetic Instructions - Negate
+    Page 72 of IBM Power ISATM Version 3.0 B
+    example:
+    7c 22 00 d0   neg  r1, r2
+    7c 22 00 d1   neg. r1, r2 *)
+let neg cpu ops =
```

Comment:
can you please explain how you are handling the dots? I don't see any CRx specific code here, though you're writing in the comments that this code corresponds to both `neg` and `neg.`. 

The same question for all other dot instructions.